app/src/main/AndroidManifest.xml

```diff
@@ -17,6 +17,11 @@
         <activity
             android:name=".AccountInformation"
             android:exported="true" >
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+
+                <category android:name="android.intent.category.LAUNCHER" />
```

Comment:
Don't forget to delete the LAUNCHER, because there should only be one.

Reply:
Done